doc/nrf/shortcuts.txt

```diff
@@ -53,7 +53,7 @@
    When enabled, it allows devices to manage transmission by informing each other about their current state, and ensures more reliable connection in high-speed communication scenarios.
 
 .. |zigbee_version| replace:: Zigbee 3.0
-.. |zboss_version| replace:: 3.6.0.0
+.. |zboss_version| replace:: 3.8.0.1
 .. |zigbee_ncp_package_version| replace:: 0.9.5
```

Comment:
We need to update this beauty before this PR is merged.

Reply:
Updates